packages/ui/booker/DatePicker.tsx

```diff
@@ -94,7 +96,13 @@ const Days = ({
           ) : (
             <DayComponent
               date={day}
-              onClick={() => props.onChange(day)}
+              onClick={() => {
+                props.onChange(day);
+                window.scrollTo({
+                  top: 360,
+                  behavior: "smooth",
```

Comment:
we could do something like "isMobile" to only scroll on mobile, however given the desktop view has more than 360px height it doesn't scroll.

it still runs "scrollTo" as an invisible operation but it's a very light JS API so I doubt we need to check if it's mobile or not

Reply:
@emrysal NIT: worth extracting into a named function?

Reply:
For a single use this is fine, keep in mind window isn't always available - but it looks OK in this situation